whatis277/harvest
# *: Fix public module name

Initially, harvest was made to be private, not public.

Now that the repo is public, it's helpful to use the correct module naming pattern.

`harvest` -> `github.com/whatis277/harvest`

Adding the package will be done in the future, once we have shared packages under `pkg/*`.

Testing instructions:
1. `dc up --build bean`
2. `dc exec -e INTEGRATION=1 bean go test ./...`
3. Visit http://localhost:8080/
4. Ensure all is successful with no errors
